sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -432,6 +432,23 @@ trait CheckAnalysis extends PredicateHelper with LookupCatalog {
 
             create.tableSchema.foreach(f => TypeUtils.failWithIntervalType(f.dataType))
 
+          case create: V2CreateTablePlanX =>
```

Comment:
This is only temporary and will be removed after `CreateTableAsSelect`, `ReplaceTable` and `ReplaceTableAsSelect` are migrated to v2 command framework.